File: slack_to_somewhat/history_try_but_error/es6/slack_hook/websocket/const/response.js
export const STATUS_CODE = {
  SUCCESS: 200,
  ERROR: 500
}

export const successRes = (message = 'success') => {
  return {
    statusCode: STATUS_CODE.SUCCESS,
    message: message
  }
}

export const errorRes = (message = 'error') => {
  return {
    statusCode: STATUS_CODE.ERROR,
    message: message
  }
}

File: rails/slack_to_somewhat/slack_to_backlog/app/models/concerns/response.rb
module Response
  extend ActiveSupport::Concern

  included do
    def success
      { status_code: 200, message: 'success' }
    end

    def error(message = C.api.external.response.error.internal_server_error)
      { status_code: 500, message: message }
    end
  end
end
